roles/hci_prepare/templates/dpservice-nova-custom-ceph.yml.j2

```diff
@@ -20,4 +20,5 @@ spec:
     networks:
     - ctlplane
     issuer: osp-rootca-issuer-internal
+    edpmRoleServiceName: nova
```

Comment:
note that this requries a crd chnage which will not be backported to beta
so this patch should not be cherry-picked to beta either.

Reply:
this does look correct however althoug hwe may end up renaming or moving this again in the future.
assuming https://review.rdoproject.org/zuul/stream/7135c8022b7840218dbecbce35c59d26?logfile=console.log passes then i think we can proceed with this to unblock ci.

Reply:
looking at 

the output this only modifies the ceph specific version

the "default" nova-custom service does not define the edpmRoleServiceName: nova field

2024-05-23 12:36:49.367446 | controller |     - apiVersion: dataplane.openstack.org/v1beta1
2024-05-23 12:36:49.367450 | controller |       kind: OpenStackDataPlaneService
2024-05-23 12:36:49.367454 | controller |       metadata:
2024-05-23 12:36:49.367458 | controller |         labels:
2024-05-23 12:36:49.367463 | controller |           created-by: install_yamls
2024-05-23 12:36:49.367467 | controller |         name: nova-custom
2024-05-23 12:36:49.367471 | controller |         namespace: openstack
2024-05-23 12:36:49.367475 | controller |       spec:
2024-05-23 12:36:49.367480 | controller |         caCerts: combined-ca-bundle
2024-05-23 12:36:49.367485 | controller |         configMaps:
2024-05-23 12:36:49.367491 | controller |         - nova-extra-config
2024-05-23 12:36:49.367497 | controller |         containerImageFields:
2024-05-23 12:36:49.367503 | controller |         - NovaComputeImage
2024-05-23 12:36:49.367508 | controller |         - EdpmIscsidImage
2024-05-23 12:36:49.367514 | controller |         playbook: osp.edpm.nova
2024-05-23 12:36:49.367519 | controller |         secrets:
2024-05-23 12:36:49.367523 | controller |         - nova-cell1-compute-config
2024-05-23 12:36:49.367528 | controller |         - nova-migration-ssh-key


so we probably need to tweak 

https://github.com/openstack-k8s-operators/install_yamls/blob/4f8ade9351e16f9acda19c139808765dde447a79/scripts/gen-nova-custom-dataplane-service.sh#L20-L33

or update

https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml

which would be the better change to include edpmRoleServiceName: nova as well.

what you proposed should be enough for the ceph job